### ISSUE TITLE
-Prelease.overriddenBranchName= leads to version ending with "-"

### ISSUE THREAD

**Issue by david0** [allegro/axion-release-plugin#480]
Hi,
we use the plugin on gitlab CI and since its doing a detached HEAD checkout, we use the following command:

```
 ./gradlew publish -Prelease.overriddenBranchName=${CI_COMMIT_BRANCH} 
 ```

Gitlab CI does a run for the newly created tag too, for this `CI_COMMIT_BRANCH` is empty/unset. The plugin will then create a version ending with `-`, e.g. `1.0.7-`. Surely we can if/else that in the job script, but it would be way more convenient if axion would tread an empty string the same as not specifying `-Prelease.overriddenBranchName` at all

Would it an option to make `-Prelease.overriddenBranchName=""` behave the same as not using overriddenBranchName at all?

#443

**Comment by bgalek** [comment]
@david0 seems reasonably!

**Comment by bgalek** [comment]
@david0 do you want to create a PR with this change? should be easy ;)

**Comment by bgalek** [comment]
merged, will be released soon!

**Comment by bgalek** [comment]
Released! Check out 1.14.0! @david0
